source/extensions/common/wasm/wasm.cc

```diff
@@ -186,7 +186,9 @@ Word resolve_dns(void* raw_context, Word dns_address_ptr, Word dns_address_size,
     root_context->onResolveDns(token, status, std::move(response));
   };
   if (!context->wasm()->dnsResolver()) {
-    context->wasm()->dnsResolver() = context->wasm()->dispatcher().createDnsResolver({}, false);
+    const auto dns_resolver_options = envoy::config::core::v3::DnsResolverOptions();
+    context->wasm()->dnsResolver() =
+        context->wasm()->dispatcher().createDnsResolver({}, dns_resolver_options);
```

Comment:
We can probably consolidate these into a single statement.  The new variable will probably get optimized out.  It's your call.

Reply:
Yes, better to consolidate this as you suggest. I was wondering what is the best practice. Now I moved the DnsResolverOptions init as the function call argument.